CLIENTS.md

```diff
@@ -0,0 +1,75 @@
+# Featured Client Requirements
+Authored by: ShowierData9978 <contact@showierdata.xyz> on 4/18/24 (CT).
+
+## Terms/Definitions
+
+- Us/We - The Meower team
+
+## Rules
+- Clients created for malicious intent will not be featured
+- If your client implements any kind of plugin/addon interface, any plugins that are featured (or in the repo) are considered to be part of the client.
+- The Meower Team may, at it's sole discretion, choose whether or not a client should be featured
+- The Meower Team may feature a client despite not complying with this RFC
+
+## Guidelines
+
+### Design Practices
+
+- Cache chat and post data
+- Use Cloudlink to get updates to cached information
+- You should use the latest version of the API
+
+### Vulnerabilities
+
+If your client is discovered to have a vulnerability, you must fix said vulnerability as soon as possible, alert your client's users (if necessary), and have it removed from the list of featured clients if it won't be fixed in a reasonable amount of time.
+
+
+### Feature Parity
+To be featured, your client must have at least these features. 
+
+- Sending, editing, and deleting posts
+- Updating chats when `update_chat` is sent (or similar in the case of updates) 
+- User profiles
+	- Quotes
+- Profile picture support
+- Post/user reporting
+- Chats:
+    - Chat creation
+    - Chat member management
+- Logging in
+- Signing up
+- Home page
+- Settings
+	 - Quote
+ 	- Typing indicator
+- Post reactions
```

Comment:
undocumented. (still have no comprehension how to properly post a reaction, api.meower.org/docs doesnt help at all)

Reply:
documentation is out of scope

Reply:
why is that?

why then make a requirements page if the developers don't even know how to use your tools? this is just confusing, why is making a documentation for other developers to actually understand your servers is less important than continuing making it more complex and harder to make them more confused and making a documentation harder in the end?